src/tasks/LargeFileUploadTask.ts

```diff
@@ -146,10 +150,21 @@ export class LargeFileUploadTask {
 	 */
 	public constructor(client: Client, file: FileObject, uploadSession: LargeFileUploadSession, options: LargeFileUploadTaskOptions = {}) {
 		this.client = client;
-		this.file = file;
+
+		if (!file.sliceFile) {
+			console.warn("Please pass a fileObject instance");
```

Comment:
This checks if the sliceFlice function property is present. This check applies on a JS code as it is not typed and does not have interfaces.  As sliceFile is a new property in the existing `FileObject` interface, this will throw a runtime error if the JS code uses the old `FileObject` structure.